processor/transform/readme-components.ts

```diff
@@ -64,6 +64,26 @@ const coerceJsxToMd =
         },
       };
 
+      parent.children[index] = mdNode;
+    } else if (node.name === 'HTMLBlock') {
+      const { position } = node;
+      const children = getChildren<HTMLBlock['children']>(node);
+      const { runScripts } = getAttrs<Pick<HTMLBlock['data']['hProperties'], 'runScripts'>>(node);
+      const html = formatHTML(children.map(({ value }) => value).join(''));
```

Comment:
I think trimming the content is fine, but I worry that changing the indent could be problematic. I'm inclined to say we shouldn't munge the string beyond trimming leading and trailing whitespce?

Reply:
yeah i'm not a big fan of the munging, which is why i tried to keep it to a minimum. if you just `.trim()`, the raw HTML displays in a messy way in the widget code editor/the recompiled markdown :(

Reply:
What kind of messy way, cause it seems wrong to me: https://github.com/readmeio/markdown/pull/910/files#diff-3abd1a08f8ea2dfcc23db22057e28a7278deda6fb0f0e81583b1babe581edf76R17-R29

Reply:
it reindents the HTML so it's not at the same indentation as the main `HTMLBlock` component. i admit it was a subjective style choice, but it helps make the md doc easier to read?

Reply:
(while also keeping the original structure as much as possible)

Reply:
(and i totally admit that there are a few scenarios here i need to account for)